### PR TITLE
add error code for unsafe safepoint

### PR DESCRIPTION
**Pull request by atmzhou**
add an error code for unsafe safepoint

### CODE REVIEW

File: sql/error.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ TiDB 兼容 MySQL 的错误码，在大多数情况下，返回和 MySQL 一样
 | 9003 | TiKV 操作繁忙，一般出现在数据库负载比较高时，请检查 TiKV Server 状态/监控/日志 |
 | 9004 | 当数据库上承载的业务存在大量的事务冲突时，会遇到这种错误，请检查业务代码 |
 | 9005 | 某个 Raft Group 不可用，如副本数目不足，出现在 TiKV 比较繁忙或者是 TiKV 节点停机的时候，请检查 TiKV Server 状态/监控/日志 |
+| 9006 | GC 间隔时间过短，长事务本应读到的数据可能被清理了 |
```
Comment (shenli) [inline sql/error.md]: 这里可以说一下如何解决，或者是链接到其他的文档中

**Comment by disksing** [comment]
LGTM.

File: sql/error.md (hunk 1 above)
Comment (sunhao2017) [reply sql/error.md]: @atmzhou Please address the comment.

**Comment by buggithubs** [comment]
@lilin90

**Pull request merged**